metadata.json

```diff
@@ -59,15 +59,15 @@
   "requirements": [
     {
       "name": "pe",
-      "version_requirement": "3.x"
+      "version_requirement": ">= 3.0.0 < 5.0.0"
     },
     {
       "name": "puppet",
-      "version_requirement": "3.x"
+      "version_requirement": ">= 3.0.0 < 5.0.0"
     }
   ],
   "dependencies": [
-    {"name":"puppetlabs/stdlib","version_requirement":"4.x"},
+    {"name":"puppetlabs/stdlib","version_requirement":">= 4.0.0 < 6.0.0"},
```

Comment:
This is because Puppet v3 is compatible with stdlib v3 - v5

https://github.com/puppetlabs/puppetlabs-stdlib#version-compatibility